docs/source/build-run-openvino.md

```diff
@@ -0,0 +1,202 @@
+# Building and Running ExecuTorch with OpenVINO Backend
+
+In this tutorial we will walk you through the process of setting up the prerequisites, building OpenVINO backend library, exporting `.pte` models with OpenVINO optimizations, and executing the exported models on Intel hardware.
+
+<!----This will show a grid card on the page----->
```

Comment:
I was not able to see the grid on the page. Not sure if this is a syntax issue or something wrong on my side. Sharing the screenshot from my side:
![image](https://github.com/user-attachments/assets/afd6a273-2ce5-4219-9924-c079e5da04ed)